Makefile

```diff
@@ -31,12 +31,13 @@ TEST = go test $(TEST_PKGS) $(TESTARGS)
 VET_PKGS ?= $(GOTARGET)/...
 VET = go vet $(VET_PKGS)
 
-DOCKER_TEST ?= $(DOCKER) run --rm -v $(DIR):$(BUILDMNT) -w $(BUILDMNT) $(BUILD_IMAGE) /bin/sh -c
+DOCKER_BUILD ?= $(DOCKER) run --rm -v $(DIR):$(BUILDMNT) -w $(BUILDMNT) $(BUILD_IMAGE) /bin/sh -c
 
 all: container
 
 container:
-	$(DOCKER) build -t $(REGISTRY)/$(TARGET):latest --target release -t $(REGISTRY)/$(TARGET):$(VERSION) .
+	$(DOCKER_BUILD) 'go build'
```

Comment:
We need to be building the binary for 32-bit if we are going to be running it on alpine:3.9
```
Linux b789bf130d06 4.9.184-linuxkit #1 SMP Tue Jul 2 22:58:16 UTC 2019 x86_64 Linux
```

I think moving the binary into `/app` is also a good idea so its not in root, but then the binary needs to be the right architecture.

Reply:
adding `CGO_ENABLED=0` to `go build` fixed the issue.